stl/CMakeLists.txt

```diff
@@ -406,7 +406,7 @@ include_directories(BEFORE
     "${TOOLSET_ROOT_DIR}/crt/src/vcruntime"
     )
 
-function(add_stl_dlls D_SUFFIX THIS_CONFIG_DEFINITIONS THIS_CONFIG_COMPILE_OPTIONS GL_FLAG THIS_CONFIG_LINK_OPTIONS)
+function(add_stl_dlls D_SUFFIX THIS_CONFIG_DEFINITIONS THIS_CONFIG_COMPILE_OPTIONS GL_FLAG THIS_CONFIG_LINK_OPTIONS TARGET_NAME)

```

Comment:
I don't strictly need this change anymore. Should I drop it?

Reply:
I can imagine these targets being useful, but it may make sense to split them into a separate PR.